SDWebImage/SDWebImageDecoder.m

```diff
@@ -46,11 +46,18 @@ + (UIImage *)decodedImageWithImage:(UIImage *)image {
         bitmapInfo |= kCGImageAlphaPremultipliedFirst;
     }
 
+    // iOS does not support 16-bit components or floating point components
+    size_t bitsPerComponent = CGImageGetBitsPerComponent(imageRef);
+    if (bitsPerComponent > 8) {
+        bitsPerComponent = 8;
+    }
+    infoMask &= ~kCGBitmapFloatComponents;
```

Comment:
maybe i am missing some thing but i don't see infoMask being used beyond this line..so what is the point of this line? is it supposed to be bitmapInfo instead?